### PR TITLE
NodeJS and yarn updates

### PR DESCRIPTION
**Pull request by rahearn**
## Description of change

* Updates nodejs to the most recent v14 LTS patch directly supported by cloud.gov-supplied buildpack
* Runs `yarn upgrade` to get all updates that satisfy version requirements in package.json (front and back end)
* Updates known vulnerabilities files

## Issue(s)

* none

### Every PR

~- [ ] Meets issue criteria~
~- [ ] JIRA ticket status updated~
- [x] Code is meaningfully tested
~- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)~
- [x] API Documentation updated
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 applications:
   - name: tta-smarthub-((env))
     buildpacks:
-      - https://github.com/cloudfoundry/nodejs-buildpack.git#v1.7.54
+      - nodejs_buildpack
     env:
```
Comment (thewatermethod) [inline manifest.yml:6]: @SarahJaine or @jasalisbury will be able to tell me if I'm imagining things, but I think I remember them updating this recently from 'nodejs_buildpack', something to do with CI failures.

Comment (rahearn) [reply manifest.yml:6]: `nodejs_buildpack` means "the version of the buildpack that cloud.gov supplies". The version can be found by running `cf buildpacks | grep nodejs` in a terminal. And the versions of node supported by that buildpack can be found by looking the version up here: https://github.com/cloudfoundry/nodejs-buildpack/releases

The pro of using this is that we're more in line with the SSP that our base image comes from cloud.gov as opposed to from cloud foundry without the cloud.gov review.

The downside is that sometimes a new version of the buildpack is released that doesn't have NodeJS version overlap with what the app is using, so deploys will fail until you either pin the buildpack version (such as to 1.7.54, here) or update the NodeJS version in use to something in the official buildpack, as this PR does.

Comment (thewatermethod) [reply manifest.yml:6]: gotcha, so moving back to the stock buildpack moves the onus on us to keep node updated to the latest supported version.

**Review by thewatermethod — APPROVED** [review]

File: manifest.yml (hunk 1 above)
Comment (rahearn) [reply manifest.yml:6]: Yes, and/or switching back to a tagged version if need be. But in the meantime there's the benefit of using the officially blessed version.

**Pull request merged**